httpmw/operationid/operationid.go

```diff
@@ -0,0 +1,92 @@
+package operationid
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"google.golang.org/grpc/metadata"
+)
+
+type contextKey string
```

Comment:
There is no need to redefine the type. The keys are in metadata, not in context.

Reply:
Right, done